wolfcrypt/src/ecc.c

```diff
@@ -3289,6 +3296,43 @@ int wc_ecc_get_curve_id_from_params(int fieldSize,
     return ecc_sets[idx].id;
 }
 
+int wc_ecc_get_curve_id_from_dp_params(const ecc_set_type* dp)
```

Comment:
Please add an API unit test (tests/unit.c) for this new API. Also make sure it gets added to `doc/dox_comments/header_files/ecc.h`.